src/copilot/chatAgentRequestHandler.ts

```diff
@@ -1054,3 +1088,51 @@ export const createSqlAgentRequestHandler = (
 
     return handler;
 };
+
+export const provideFollowups = (
```

Comment:
Nit: more readable to use `function` syntax instead of `const` syntax whenever possible.